cli/go.mod

```diff
@@ -18,25 +20,25 @@ require (
 	github.com/posener/complete v1.2.3
 	github.com/stretchr/testify v1.8.4
 	golang.org/x/text v0.14.0
-	helm.sh/helm/v3 v3.11.3
-	k8s.io/api v0.26.12
-	k8s.io/apiextensions-apiserver v0.26.10
-	k8s.io/apimachinery v0.26.12
-	k8s.io/cli-runtime v0.26.10
-	k8s.io/client-go v0.26.12
-	k8s.io/utils v0.0.0-20230220204549-a5ecb0141aa5
+	helm.sh/helm/v3 v3.14.4
```

Comment:
This is the `helm/v3` update that resolves a couple of CVEs (see included changelog)